.github/workflows/e2e.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         run: brew install coreutils sponge
       - uses: cargo-bins/cargo-binstall@main
       - name: Install cargo-dist
-        run: cargo binstall cargo-dist@0.4.3 -y
+        run: cargo binstall cargo-dist@0.9.0 -y
```

Comment:
heads up: .github/workflows/release.yml is nonstandard.  The `plan` check isn't happening in this PR, so you may run into problems on the next PR, or when next trying to release.